### ISSUE TITLE
Encountering `CERTIFICATE_VERIFY_FAILED`

### ISSUE THREAD

**Issue by omggodlike123** [doronz88/pymobiledevice3#991]
**Test environment**
- Host OS version: mac 10.15
- Target device model and iOS version:  ios 17.4.1

**Describe the bug**
already open develop options on iphone, when i try to mount DeveloperDiskImage, get a ssl error

**To Reproduce**
open developer options and mounter the DeveloperDiskImage 

**Logs**
mounte device img failed, error: 2024-05-12 21:57:33 nwooden--MacBookPro18 pymobiledevice3.cli.mounter[4449] WARNING failed to query DeveloperDiskImage versions: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)

## For community
⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

**Comment by omggodlike123** [comment]
python don't support ssl on mac?  @doronz88 @davesc63

**Comment by rob-X1** [comment]
To my knowledge iOS 17.x doesn't use DeveloperDiskImages anymore. Xcode also has no developer disk images for 17.x devices.

**Comment by doronz88** [comment]
Sound like you're having a general SSL errors. Try re-pairing or fixing the device/you-mac's datetime settings

**Comment by omggodlike123** [comment]
> Sound like you're having a general SSL errors. Try re-pairing or fixing the device/you-mac's datetime settings

also don't work

looks like the same problem like https://stackoverflow.com/questions/27835619/urllib-and-ssl-certificate-verify-failed-error/73270162#73270162

**Comment by doronz88** [comment]
do any other commands work? such as the `syslog live`?

**Comment by omggodlike123** [comment]
> do any other commands work? such as the `syslog live`?

it works. only have problem when i try use mounter on mac

**Comment by doronz88** [comment]
Could not reproduce. Please try to debug yourself and report back where is this generated at

**Comment by omggodlike123** [comment]
this bug happenes when i use mounter query DeveloperDiskImage versions
it will reproduce when a user's mac don't have ssl CERTIFICATE.

**Comment by omggodlike123** [comment]
> this bug happenes when i use mounter query DeveloperDiskImage versions
> it will reproduce when a user's mac don't have ssl CERTIFICATE.

this bug happenes when i use mounter query DeveloperDiskImage versions
it will reproduce when a user's mac don't have ssl CERTIFICATE.

**Comment by doronz88** [comment]
I don't understand your answer. Anyhow a PR is welcome

**Comment by doronz88** [comment]
Closing — CERTIFICATE_VERIFY_FAILED is a host-side missing-CA-certificates problem (only affects the mounter HTTP query) and was not reproducible as a library bug.
